packages/cicero-cli/package.json

```diff
@@ -46,11 +46,12 @@
     "tmp-promise": "1.0.4"
   },
   "dependencies": {
-    "@accordproject/cicero-core": "0.3.14",
-    "@accordproject/cicero-engine": "0.3.14",
+    "@accordproject/cicero-core": "0.4.0",
+    "@accordproject/cicero-engine": "0.4.0",
     "@accordproject/ergo-compiler": "0.0.48",
```

Comment:
The version of the ergo-compiler currently used by Cicero is 0.0.62 (https://github.com/accordproject/cicero/blob/52c844f0971ce5ff0cec1d99e69b7e46d262170b/packages/cicero-cli/package.json#L51) a merge issue, or was there any problem with the recent version?

Reply:
Could be a merge issue. I'll update it.

Reply:
Now that the merge has been resolved, it seems to be correct.